timeout/timeout.c

```diff
@@ -449,13 +452,14 @@ struct TClock validate_duration(char *timestr) {
  * struct is set.
  */
 int parse_interval(const char *ss, struct TClock *interval) {
-	char s[32];
```

Comment:
Why `{0}` was added?

Reply:
Let me see...

Reply:
That is meant to initialise the string, not a big deal too.

If you have another suggestion of what to use instead of ``(char)0``, I'm all ears.